mlflow/mistral/autolog.py

```diff
@@ -0,0 +1,57 @@
+import inspect
+import logging
+
+import mlflow
+import mlflow.mistral
+from mlflow.entities import SpanType
+from mlflow.mistral.chat import convert_message_to_mlflow_chat, convert_tool_to_mlflow_chat_tool
+from mlflow.tracing.utils import set_span_chat_messages, set_span_chat_tools
+from mlflow.utils.autologging_utils.config import AutoLoggingConfig
+
+_logger = logging.getLogger(__name__)
+
+
+def construct_full_inputs(func, *args, **kwargs):
```

Comment:
nit: this could be "_" prefixed method (sorry it was initially my overlook).

Reply:
updated